### DISCUSSION TITLE
SharePoint localpeoplesearch

### DISCUSSION THREAD

**Discussion by gdanskii** [microsoft-search/pnp-modern-search#2190] [Q&A]
Hi All

Very new to this, I created the search and appear to work ok, returns people ok, see image.  However with other searchs, i can click on person and it takes me to the profile page, this doesn't do that, is it something i can do and if so how?

Thanks

Jamie

![image](https://user-images.githubusercontent.com/108523599/176876566-44c212c5-e5f1-420c-bc6f-e4fab5a6dcb1.png)

**Comment by wobba** [comment]
In theory you should be able to add markup to the People Fields layout options, but seems it's not persisted. @FranckyC any idea?

**Comment by gdanskii** [comment]
Works if i use the list view, but not if i use the people view
